runtime/nls/j9vm/j9vm_ca.nls

```diff
@@ -1704,10 +1704,6 @@ J9NLS_VM_ILLEGAL_ACCESS_NONSTATIC_FINAL.user_response=Modify source code to avoi
 # Fourth argument is nest host name
 J9NLS_VM_NEST_HOST_HAS_DIFFERENT_CLASSLOADER=La classe %2$.*1$s i l'amfitri\u00f3 d'imbricaci\u00f3 %4$.*3$s han de tenir el mateix carregador de classes.
 # START NON-TRANSLATABLE
-J9NLS_VM_NEST_HOST_HAS_DIFFERENT_CLASSLOADER.sample_input_1=3
```

Comment:
These lines should not be removed. Here and below, and in the subsequent files as well.